### PR TITLE
Direct login with google or linkedin

### PR DESCRIPTION
**Pull request by jackbravo**
# What does this PR do?

- removes regular login page
- uses new /login/wizeline and /login/linkedin routes in the homepage
- on auth.server.tsx exports a getAuthenticator function that allows setting the `connection` option to login directly with google or LinkedIn

**TODO:** improve the UI

# Where should the reviewer start?

- review the individual commits
- Verify that the redirection is correct according to the user's role.

# How should this be manually tested?

# Any background context you want to provide?

- depends on #235 

# What are the relevant tickets?
#230 

# Screenshots

image

# Questions

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: .github/workflows/deploy.yml
Comment (jackbravo) [inline .github/workflows/deploy.yml:1]: I hope this will reduce our `segmentation fault` issues when running our CI

**Review by AxelUZ — COMMENTED** [review]
When I log in it sends me directly to the auth0 panel and not to the Google service or Linkdin directly. Also it does not check if the applicationForm was answered or not (but I can do all this validation in my ticket). I think it is a good implementation because this greatly reduces the number of screens and facilitates navigation 👍

**Comment by CarlosMilano** [comment]
there is also a bug with the redirect, when you log in with linkedin it sends you to /projects instead of /intershipProjects

**Comment by martinrobled0** [comment]
> there is also a bug with the redirect, when you log in with linkedin it sends you to /projects instead of /intershipProjects

also noticed this

**Comment by jackbravo** [comment]
> there is also a bug with the redirect, when you log in with linkedin it sends you to /projects instead of /intershipProjects

Ah yes! This probably can also be changed, in a similar fashion to what I'm doing for the connection option, or something similar.

Same for @AxelUZ comments.

@CarlosMilano , @AxelUZ do you want to take ownership of this PR?

**Comment by AxelUZ** [comment]
> > there is also a bug with the redirect, when you log in with linkedin it sends you to /projects instead of /intershipProjects
> 
> Ah yes! This probably can also be changed, in a similar fashion to what I'm doing for the connection option, or something similar.
> 
> Same for @AxelUZ comments.
> 
> @CarlosMilano , @AxelUZ do you want to take ownership of this PR?

Sure, I will add it to my ticket and start working on it. 👍

**Review by CarlosMilano — COMMENTED** [review]
LGTM!!

**Review by martinrobled0 — APPROVED** [review]
LGTM

**Pull request merged**